eng/tox/verify_whl.py

```diff
@@ -107,16 +106,33 @@ def should_verify_package(package_name):
 
     # get target package name from target package path
     pkg_dir = os.path.abspath(args.target_package)
-    pkg_name, namespace, ver = get_package_details(os.path.join(pkg_dir, "setup.py"))
+    pkg_name, namespace, ver, python_requires = get_package_details(os.path.join(pkg_dir, "setup.py"))
+
+    top_level_module = namespace.split(".")[0]
+    wheel_location = get_wheel(args.dist_dir, ver)
 
-    top_level_module = namespace.split('.')[0]
+    if not os.getenv("IGNORE_WHEEL_PYVERSION_CHECK"):
+        if not python_requires:
+            logging.info("Your package must define a setup argument for 'python_requires'.")
```

Comment:
Can all errors message point to https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/packaging.md additionally?